### ISSUE TITLE
TypeError: Object.entries is not a function

### ISSUE THREAD

**Issue by yogeek** [ameerthehacker/kubelive#17]
**Describe the bug**
Just installed the tool with : 
```sudo npm install -g kubelive```

and when trying to use it :
```
$ export KUBECONFIG=<PATH_TO_KUBECONFIG_FILE>

$ kubectl cluster-info                           
Kubernetes master is running at ...
KubeDNS is running at ...
Metrics-server is running at ...

$ kubelive                                                                         
/usr/local/lib/node_modules/kubelive/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js:53
        throw ex;
        ^

TypeError: Object.entries is not a function
    at Module.assembleStyles [as exports] (/usr/local/lib/node_modules/kubelive/node_modules/slice-ansi/node_modules/ansi-styles/index.js:123:42)
    at Function.Module._load (module.js:440:16)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/kubelive/node_modules/slice-ansi/index.js:4:20)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```

**Expected behavior**
kubelive displays help message

**Desktop (please complete the following information):**

- OS: [e.g. iOS] Ubuntu 17.10

**Additional context**
```
$ kubectl version --short
Client Version: v1.17.0
Server Version: v1.16.6

$npm -v                               
3.5.2

```

**Comment by ameerthehacker** [comment]
Hi @yogeek, what is your node version,  can you please try upgrading to v10

**Comment by yogeek** [comment]
Hi @ameerthehacker , thank you for your help. As I indicated in my previous comment, my npm version was "3.5.2". I just tested with the latest docker image node (version v13.8.0), and it is working well.
Perhaps you can indicate the mininal node version kubelive is compatible with in the install instructions ?

And if you are interested, here are the instructions to launch your tool with docker :
```
# Build kubelive docker image
docker build -t kubelive -f - . << EOF                                                                                                                                                    
FROM node:latest
RUN npm install -g kubelive
ENTRYPOINT ["kubelive"]
CMD ["-h"]
EOF

# Use it
docker run --rm -it --net=host -v ~/.kube/kubeconfig:/root/.kube/kubeconfig -e KUBECONFIG=/root/.kube/kubeconfig kubelive get pods
```

Thanks !

**Comment by ameerthehacker** [comment]
Hi @yogeek, this is fixed by  #18 and it has been published to npm so I'm closing this issue

Please feel free to reopen the issue if you still face any issues
